CONTRIBUTING.md

```diff
@@ -69,7 +69,9 @@ After this, you should be able to run `make coverage` to run the tests and see t
 
 ## Debugging
 
-**TF_LOG=DEBUG terraform apply** allows you to see the internal logs from `terraform apply`.
+**TF_LOG_PROVIDER=DEBUG terraform apply** allows you to see the internal logs from `terraform apply`.
```

Comment:
maybe add a note that it's TF 0.15.0+ variable?

Reply:
more than 50% of customers use 1.0+, so it's safe enough. Also i don't suspect any contributors to run something older than 1.0 or 1.1